WorkMode/guiding_principles.md

```diff
@@ -0,0 +1,86 @@
+
+## Require at least two supporters for each use case.
+
+As the mission is interoperability, having at least two supporters for each use case is important. If only one supporter can be found, then any features that it would require are likely too specific for inclusion in the proposed form.
+
+**Notes**
+
+* Any independent organization, group or individual can be a supporter, regardless of whether they are part of the Working Group, or even members of the W3C. Consortia or other standards organizations that represent many participants can thus fulfill this principle via their members.
+* This is orthogonal to implementations of the specifications, which are required to exit Candidate Recommendation phase.
+
+## As simple as possible, but no simpler.
+
+A simpler solution (to understand, implement and use) is better than a more complex solution that accomplishes the same result.
+
+## Consistency is simpler than exceptions.
+
+Simplicity is considered in aggregate for the set of features defined by a specification, not independently.  If the same set of features can be accomplished by a smaller number of more consistent patterns, then that method is (very likely) simpler. Memorizing exceptions is harder than memorizing and applying rules.
+
+**Notes**
+
+* The WG discussed up to this point on 2018-07-06.
+
+## Simplicity is determined by end users, not library implementers.
+
+Simplicity (or usability) is determined by the target audience (data producers and consumers) based on their experience of understanding and applying the specification via existing implementations, not by the experience of implementers of the specification text directly. If there is a feature that makes it harder for implementations, but easier/better for end users, then that is a worthwhile trade off.
+
+## Provide on-ramps.
+
+A solution that can be implemented in incremental stages is better than a
+solution that is all or nothing, as not everyone needs every feature but
+many people need various parts.
+
+## Define success, not failure.
+
+We should define things in terms of what it means to be conformant, rather
+than what is not conformant.  The fewer constraints we require, the easier
+to have non-breaking changes in the future and the easier it is to have
+experimentation.
+
+## Follow existing standards and best practices, where possible and where they do not conflict with other principles.
+
+Between invention and reuse, pick reuse... unless that reuse would
+demonstrably harm adoption by being more complicated than necessary.
+
+## The underlying data model is RDF.
+
+If a feature comes up that can't be modeled with RDF as the underlying abstract data model, then we refer the feature to a future RDF Working Group for potential inclusion at that time. Similarly, we should ensure that the features of RDF are expressed in JSON-LD, to ensure that data can be round-tripped with confidence through different serializations.
+
```

Comment:
We need to define what we mean by “round-tripped”; does this include prefix selection, specific object embedding, use of encoding features, array ordering?

Reply:
@azaroth42 any chance we can get this clarification added?